receiver.py

```diff
@@ -84,11 +89,18 @@ def data_lines(self):
                 # Read first two bytes, to determine packet type
                 self.packet_type = int(payload[:2], 2)
 
+                #Preallocate space
+                stored_data = '' 
                 # Unpack Struct according to ID, and update global parameters
-                self.stored_data[self.packet_type] = self.data_shape[self.packet_type].unpack(payload)
+                for i in xrange(3):
+                    if (self.packet_type == i):
+                        stored_data += self.data_shape[self.packet_type].unpack(payload)
+                    else:
+                        stored_data += self.data_shape[self.packet_type].unpack(','*len(self.data_shape))
```

Comment:
This adds the "blank" spots in the telemetry. Seems a little sketch still.

As a side note, I'll change the len() into a .size later....